src/Morphir/Visual/ViewArithmetic.elm

```diff
@@ -9,8 +9,8 @@ import Morphir.Visual.EnrichedValue exposing (EnrichedValue)
 import Morphir.Visual.Theme exposing (smallPadding, smallSpacing)
 
 
-view : Config msg -> (EnrichedValue -> Element msg) -> ArithmeticOperatorTree -> Element msg
-view config viewValue arithmeticOperatorTree =
+view : Config msg -> Int -> (Int -> EnrichedValue -> Element msg) -> ArithmeticOperatorTree -> Element msg
+view config zIndex viewValue arithmeticOperatorTree =
```

Comment:
Would it be possible to include the `zIndex` in the config or derive it from the `EnrichedValue` somehow? Our goal here is to have different `zIndex` values for each drill-down level. Sounds like that information is already available given that we are tracking expanded/collapsed state per level.

Reply:
we don't just want to have drilldowns have lower z-index thank their parents, unfortunately they have to have a lover z-index than previous drilldowns on the same level of the expression, because of cases like the attached image
![image](https://github.com/finos/morphir-elm/assets/18498914/ba0cd5e3-3239-4099-9618-865ffa9910b6)